ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/util/KeystoreUtil.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.util;
+
+import io.confluent.ksql.util.KsqlException;
+import io.vertx.core.buffer.Buffer;
+import java.io.ByteArrayOutputStream;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.security.KeyStore;
+import java.security.PrivateKey;
+import java.security.cert.Certificate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public final class KeystoreUtil {
+  private static final Logger LOG = LoggerFactory.getLogger(KeystoreUtil.class);
+  private static final String KEYSTORE_TYPE = "JKS";
+
+  private KeystoreUtil() {}
+
+  /**
+   * Utility to fetch a Vert.x Buffer that is the serialized version of the key store from the
+   * given path, but which contains only the entry for the given alias.  This circumvents Vert.x's
+   * direct support of an alias option.
+   * @param keyStorePath The original key store which may contain multiple certificates
+   * @param password The key store password
+   * @param alias The alias of the entry to extract
+   * @return The Buffer containing the keystore
+   */
+  public static Buffer getKeyStore(
+      final String keyStorePath,
+      final String password,
+      final String alias
+  ) {
+    final char[] pw = password != null ? password.toCharArray() : null;
```

Comment:
Better to encode the optionality of `password` into the signature of the method? i.e. `Optional<String> password`?